NoClassDefFoundError exception when transforming XML to PDS3 label

### Checked for duplicates

No - I haven't checked

### 🐛 Describe the bug

I downloaded transform and followed instructions related to setting of paths etc.  The output of transform -h was as expected, but I got stymied with a "missing class"  exception: 

cfassett@FASSECI1-WL1:/mnt/d/Projects/Ch2-ohrc/ch2_ohr_ncp_20211223T0019163816_d_img_d32/data/calibrated/20211223$ transform -f pds3-label ch2_ohr_ncp_20211223T0019163816_d_img_d32.xml
Exception in thread "main" java.lang.NoClassDefFoundError: com/sun/xml/bind/v2/model/annotation/AnnotationReader
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:756)
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
        at java.net.URLClassLoader.defineClass(URLClassLoader.java:473)
        at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        at java.lang.Class.getDeclaredMethods0(Native Method)
        at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
        at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
        at java.lang.Class.getMethod0(Class.java:3018)
        at java.lang.Class.getMethod(Class.java:1784)
        at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:169)
        at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:131)
        at javax.xml.bind.ContextFinder.find(ContextFinder.java:335)
        at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:431)
        at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:394)
        at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:298)
        at gov.nasa.pds.objectAccess.ObjectAccess.getJAXBContext(ObjectAccess.java:183)
        at gov.nasa.pds.objectAccess.ObjectAccess.getProduct(ObjectAccess.java:199)
        at gov.nasa.pds.objectAccess.ObjectAccess.getProduct(ObjectAccess.java:189)
        at gov.nasa.pds.transform.product.Pds4ToPds3LabelTransformer.getFileAreas(Pds4ToPds3LabelTransformer.java:878)
        at gov.nasa.pds.transform.product.Pds4ToPds3LabelTransformer.transform(Pds4ToPds3LabelTransformer.java:136)
        at gov.nasa.pds.transform.product.Pds4LabelTransformer.transform(Pds4LabelTransformer.java:81)
        at gov.nasa.pds.transform.product.DefaultTransformer.transform(DefaultTransformer.java:116)
        at gov.nasa.pds.transform.TransformLauncher.doTransformation(TransformLauncher.java:428)
        at gov.nasa.pds.transform.TransformLauncher.processMain(TransformLauncher.java:461)
        at gov.nasa.pds.transform.TransformLauncher.main(TransformLauncher.java:475)
Caused by: java.lang.ClassNotFoundException: com.sun.xml.bind.v2.model.annotation.AnnotationReader
        at java.net.URLClassLoader.findClass(URLClassLoader.java:387)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        ... 33 more

This was correctable on my system by explicitly editing the **bin/transform** script to call the classpath:

`${JAVA_CMD} -classpath $LIB_DIR -Xms256m -Xmx1024m -Dcom.sun.media.jai.disableMediaLib=true -Doverwrite.output=true -Dexternal.programs.home=${PARENT_DIR}/external-programs -jar ${TRANSFORM_JAR} "$@"`

which solved the problem.  I'm a python guy, so I'm not sure the environment/class variables work only this way on my system, but it might be something to alert people to who run into in the future

### 🕵️ Expected behavior

expected it to run, got errors except for explicit class call (see bug description)

### 📜 To Reproduce

_No response_

### 🖥 Environment Info

- transform 1.12.1, also happened in 1.11.5
- Operating System: Ubuntu (on Windows System for Linux)


### 📚 Version of Software Used

_No response_

### 🩺 Test Data / Additional context

_No response_

### 🦄 Related requirements

_No response_

### ⚙️ Engineering Details

_No response_

--- a/pom.xml
+++ b/pom.xml
@@ -248,12 +248,12 @@ POSSIBILITY OF SUCH DAMAGE.
     <dependency>
       <groupId>com.sun.xml.bind</groupId>
       <artifactId>jaxb-impl</artifactId>
-      <version>2.2.11.jbossorg-1</version>
+      <version>2.3.9</version>
     </dependency>
     <dependency>
       <groupId>javax.xml.bind</groupId>
       <artifactId>jaxb-api</artifactId>
-      <version>2.4.0-b180830.0359</version>
+      <version>2.3.1</version>
     </dependency>
     <!-- See the JBoss repo for jai-core and jai-codec -->
     <dependency>

